cdap-explore/src/test/java/co/cask/cdap/explore/service/ExploreServiceTestsSuite.java

```diff
@@ -31,12 +33,17 @@
   HiveExploreServiceTestRun.class,
   WritableDatasetTestRun.class,
   HiveExploreObjectMappedTableTestRun.class,
-  HiveExploreServiceStopTestRun.class
+  HiveExploreServiceFileSetTest.class
```

Comment:
The JUnit docs don't say anything about excluding Test files that are part of TestSuite from outside the suite. But I see that our config is setup that way. I'll fix the class names.